tests/Module/Order/MetaboxesWrapperTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare( strict_types=1 );
+
+namespace Tests\Module\Order;
+
+use Packetery\Core\Entity\Order;
+use Packetery\Module\Order\CustomsDeclarationMetabox;
+use Packetery\Module\Order\Metabox;
+use Packetery\Module\Order\MetaboxesWrapper;
+use Packetery\Module\Order\Repository;
+use PHPUnit\Framework\TestCase;
+use WC_Order;
+
+class MetaboxesWrapperTest extends TestCase {
+	public function testBeforeOrderSaveProcessesOrderOnce(): void {
+		$generalMetabox           = $this->createMock( Metabox::class );
+		$customDeclarationMetabox = $this->createMock( CustomsDeclarationMetabox::class );
+		$orderRepository          = $this->createMock( Repository::class );
+		$metaboxesWrapper         = new MetaboxesWrapper( $generalMetabox, $customDeclarationMetabox, $orderRepository );
```

Comment:
Řešil bych to jako v ostatních testech metodou createMetaboxesWrapper a mocky v properties.